assets/openshift-apiserver/openshift-apiserver-configmap.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: openshift-apiserver
 data:
   aggregator-client-ca.crt: |-
-    {{ include_pki "root-ca.crt" 4 }}
+{{ include_pki "root-ca.crt" 4 }}
```

Comment:
did you mean to change this? I think it will break our current automation

Reply:
Yes I did... it was not formatting the included file correctly (4 spaces indent in the first line)